Assets/HoloToolkit/Input/Tests/Scripts/SimpleGridGenerator.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class SimpleGridGenerator : MonoBehaviour
```

Comment:
Ah, I think the real underlying issue isn't whether the objects were created dynamically, but instead the current inability to dynamically decide what object to invoke from a voice command. I believe solving the latter solves the former and demonstrating it with placed objects would be fine.

I believe as you do that we're likely to add more kinds of these helper strategies for command forwarding and think that by keeping the tests to the minimum required for a solution we'll have less to manage going forward. If others find value in the dynamic allocation I'm OK with keeping it, I just don't think it's necessary.